# This service allows the user to tell the vision system to only look for a
# specific object within its library. If this option is not set, the vision
# system will try to match each type of object with the given point cloud,
# and pick the best match

bool use_object   # Whether or not to only look for a specific object
int64 objNum      # The object number in the library to look at

----

int64 ret
string msg
